### ISSUE TITLE
Structure of results.pth

### ISSUE THREAD

**Issue by elena-orlova** [SsnL/dataset-distillation#25]
Hello!

Very interesting paper, thanks! 
I have a question about the ‘’’results.pth’’’. It’s a file with distilled images (tensors), but what about its structure? For example, for MNIST its ‘’’len’’’ 30, so it’s because we have 3 steps? And labels, it’s just tensor from 0 to 9, so the order is important, like the first 3 tensors are class 0, right? 
And at the testing phase we use pretrained model which we got after distill basic? 

Thanks!

**Comment by SsnL** [comment]
Hi, 30 represents 10 steps repeated for 3 epochs. Each element of the list contains (data, label, lr).

In test phase, the model is randomly initialized / loaded, depending on the setting. distill_basic actually does not train models. It only distills images.

**Comment by elena-orlova** [comment]
Thanks a lot for your reply!
